sdk/tables/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+cmake_minimum_required (VERSION 3.13)
+
+project (azure-data LANGUAGES CXX)
```

Comment:
Should the C++ project name be `azure-tables`?

**Looking at others in the repo, we have:**
- azure-storage
- azure-security-keyvault
- azure-security-attestation
- azure-messaging
 
Also `azure-data-tables` is an option, if we don't ever expect any other sub-projects except azure-data-tables? Otherwise, it would conflict a bit (attestation has that issue atm).

Reply:
honestly no matetr what  i choose it will be brought up in a stand up , even to reach the same point where we left, so i odon't see the point in making any changes as they will be rehashed anyway . i'll await the team meedting for this.